.github/workflows/serverless-performance.yml

```diff
@@ -0,0 +1,26 @@
+name: Serverless performance test
+
+on:
+  pull_request:
+  push:
+    branches: [master]
+
+jobs:
+  performance-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout dd-trace-js
+        uses: actions/checkout@v3
+      - name: Checkout datadog-lambda-js
+        uses: actions/checkout@v3
+        with:
+          repository: DataDog/datadog-lambda-js
+          # TODO remove hardcoded branch when merged to main
+          ref: refs/heads/maxday/add-git
+          path: datadog-lambda-js
+      - name: Update package.json to the current ref
+        # TODO node_version hardcoded for now, switch to matrix for all supported version of node
+        run: |
+          cd datadog-lambda-js
+          yarn add https://github.com/DataDog/dd-trace-js#refs/heads/$GITHUB_HEAD_REF --save-dev
```

Comment:
If you're checking out dd-trace-js anyway, why not just `yarn link`?

Reply:
I tried but since it's built in a Docker container, files are copied into the container and the link is not persisted.
I could do the link in `datadog-lambda-js` but I prefer this as it's more explicit in this context?
WDYT?

Reply:
What if you did a `COPY` of the files on VM disk into the container followed by `yarn link`? 

```Dockerfile
COPY /path/to/datadog-lambda-js /tmp/datadog-lambda-js
WORKDIR /tmp/datadog-lambda-js
RUN yarn link
WORKDIR /path/to/app
RUN yarn link datadog-lambda-js
```

Reply:
We want to link `dd-trace` not `datadog-lambda-js` but yeah that would definitely work. The only thing is that this change would need to be made on the `datadog-lambda-js` repo (where the Dockerfile lives) and I would prefer not if possible. WDYT?

Reply:
It's probably fine as is. It sounds like modifying another repo adds a bunch of complexity to this CI job.

The main fear is that there's now one more activity that makes network requests as part of CI, which is one more potential place for flakiness. Of course, if we can't talk to GitHub, there's a dozen other things that will fail beforehand anyway.